rpcs3/rpcs3qt/tooltips.h

```diff
@@ -134,6 +134,7 @@ class Tooltips : public QObject
 		const QString hide_mouse_on_idle           = tr("Hides the mouse cursor if no mouse movement is detected for the configured time.");
 		const QString show_shader_compilation_hint = tr("Shows 'Compiling shaders' hint using the native overlay.");
 		const QString use_native_interface         = tr("Enables use of native HUD within the game window that can interact with game controllers.\nWhen disabled, regular Qt dialogs are used instead.\nCurrently, the on-screen keyboard only supports the English key layout.");
+		const QString enable_home_menu             = tr("When enabled, pressing the PS home button will open a menu that can be used to close the game etc..");
```

Comment:
We're having too many options for native UI.
I think we should just have 2 options:
1. Global native UI (module loading view, home menu, etc)
2. "Distracting" UI elements such as shader compiling notification and the system notifications at the top left.

Reply:
This is meant for the ps button, not for the UI itself.

Reply:
I mean, the home menu is part of native UI. I don't see a reason to ever disable it at all, its not like anyone is accidentally pressing home. It's not important, I just dislike giving the user too many options to think about.

Reply:
vsh uses it, it's the reason we added ps button handling initially.

Reply:
Then it is completely incompatible with VSH and we can manually disable intercepting in that scenario as we have all over the emulator. Why even give the user that option when the UI will hijack the input?
Also home button is broken in VSH, I forgot to push the fix for that.

Reply:
I replaced it with an option to pause emulation.
Some games may not care for the system menu, so it seems like a good idea to add that workaround.

Reply:
Home menu is VSH feature, right? Why reimplement that was already implemented?